CMakeLists.txt

```diff
@@ -24,7 +24,7 @@ TRIBITS_ADD_ETI_INSTANTIATIONS(TpetraCore
 
```

Comment:
I don't know if it is possible on github (webhooks?), but on client side a simple pre-commit hook is all you need.